mmtrack/datasets/pipelines/formatting.py

```diff
@@ -1,10 +1,11 @@
 import numpy as np
+from mmcls.datasets.builder import PIPELINES as CLS_PIPELINES
 from mmcv.parallel import DataContainer as DC
-from mmdet.datasets.builder import PIPELINES
+from mmdet.datasets.builder import PIPELINES as DET_PIPELINES
```

Comment:
We should only use a `PIPELINE`, it doesn't matter whether it is for cls/det/tracking